Backpack/Icon/Classes/BPKIconView.h

```diff
@@ -44,6 +44,20 @@ NS_SWIFT_NAME(IconView) @interface BPKIconView : UIImageView
  */
 @property(nullable, nonatomic, copy) BPKIconName iconName;
 
+/**
+ * The size of the icon currently being displayed. Setting updates
+ * the displayed icon.
+ * @see BPKIconSize
+ */
+@property(nonatomic, assign) BPKIconSize size;
+
+/**
+ * If flipsForRightToLeft is set to YES then BPKIconView will render its icon
+ * horizontally flipped when its UIUserInterfaceLayoutDirection is right to left.
+ * Default value is NO.
+ */
+@property(nonatomic, assign) BOOL flipsForRightToLeft;
```

Comment:
Why is this no by default?

Reply:
Not so sure about that and additionally it would be a breaking change

Reply:
It's no by default as most of the icons don't need to be flipped. The exceptions are the ones expressing a horizontal direction such as arrows and half stars.

Reply:
Oh yeah - sorry I didn't realise this was BPKIcon code - assumed it was for the star rating 🤦‍♂